### PR TITLE
Add blog post titled "On freenode and the move to libera.chat"

### PR DESCRIPTION
**Pull request by dequis**
Co-Authored-By: Geert Hauwaerts <geert@hauwaerts.be>

### CODE REVIEW

**Review by LemonBoy — APPROVED** [review]

**Comment by sudoforge** [comment]
> allow us to assess the possible legal consequences of removing freenode as a network in our source code.

Can this be expanded upon here or in the post itself? As a non-user of IRSSI, I'm surprised to read this line here. What legal relationship does IRSSI have with Freenode?

**Comment by dequis** [comment]
>Can this be expanded upon here or in the post itself?

No, not really. Sorry!

**Comment by hrefhref** [comment]
> > Can this be expanded upon here or in the post itself?
> 
> No, not really. Sorry!

Why ?

**Review by ahf — APPROVED** [review]
Looks good.

File: _posts/2021-05-27-on-freenode-and-the-move-to-libera-chat.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+course, extremely poorly worded. The intent was "we would have wanted to not
+have to take any political stance to begin with" - but political stances were
+taken in both cases and history can't be rewritten, neither in git nor in real
+life.
+
+We want to make it explicitly clear that the entire irssi team was always
+aligned with the general plan and shared the goal to migrate from freenode
+to another network. In no way do we endorse any recent actions taken by new
+representatives of freenode staff, nor are we associated with freenode in
+any manner besides it hosting our former primary suppport channel. However, the
```
Comment (aveao) [inline _posts/2021-05-27-on-freenode-and-the-move-to-libera-chat.markdown:33]: Typo here: "suppport"

Comment (GeertHauwaerts) [reply _posts/2021-05-27-on-freenode-and-the-move-to-libera-chat.markdown]: Fixed

**Comment by Valodim** [comment]
> No, not really. Sorry!

The only reading of this that makes any sense to me is that the irssi project does indeed have some kind of legal relationship with Freenode, and thus Andrew Lee. And worse, a legal relationship kept under uncertain terms in front of the community and contributors, not unlike Freenode's own situation for the last couple of years.

There may be a perfectly benign explanation to all of this, but those are the optics of the commit in question and this subsequent PR.

**Comment by xfix** [comment]
> > No, not really. Sorry!
> 
> The only reading of this that makes any sense to me is that the irssi project does indeed have some kind of legal relationship with Freenode, and thus Andrew Lee. And worse, a legal relationship kept under uncertain terms in front of the community and contributors, not unlike Freenode's own situation for the last couple of years.
> 
> There may be a perfectly benign explanation to all of this, but those are the optics of the commit in question and this subsequent PR.

It's possible Andrew Lee attacked irssi project with a [SLAPP](https://en.wikipedia.org/wiki/Strategic_lawsuit_against_public_participation).

**Comment by dequis** [comment]
My previous comment did not intend to imply anything but the fact that I, personally, can't give any details beyond what's in the post. Not because someone is forcing me to stay quiet, but rather I would prefer to not accidentally say stupid shit.

Ironically, even making that reply was a mistake, but, give me a break, I was and still am stressed about this mess. And it's the internet, people will speculate wildly inaccurate things regardless of what we say or not.

Anyway - since then, we expanded the post a little. Quoting the current version of the draft:

> In no way do we endorse any recent actions taken by new representatives of freenode staff, nor are we associated with freenode in any manner besides it hosting our former primary support channel

**Comment by dequis** [comment]
Also: Please keep the wild speculation out of this thread, I don't want to have to lock this one too.

**Comment by ahf** [comment]
I think we should ditch this now and remove the references to Freenode in the code. The network has killed itself entirely by now.

**Comment by dequis** [comment]
Yeah this has become irrelevant with time. Hope it provided some insight to curious bystanders, but it'd make no sense to publish this post now.

**Pull request closed without merging**